### ISSUE TITLE
Allow configuring credentials in wp-config.php

### ISSUE THREAD

**Issue by rayrutjes** [rayrutjes/wc-order-search-admin#64]
Allow the same as: https://community.algolia.com/wordpress/migration.html#ease-environment-switching-with-constants

First reported here: https://discourse.algolia.com/t/wp-config-api-for-woocommerce-order-search-admin/4402

**Comment by urlund** [comment]
@rayrutjes ... I'll suggest the following constant names to avoid a clash with the other constants, in case they - for some reason should have different values:

```
define ( 'WC_OSA_ALGOLIA_APPLICATION_ID', '<value>' );
define ( 'WC_OSA_ALGOLIA_SEARCH_API_KEY', '<value>' );
define ( 'WC_OSA_ALGOLIA_ADMIN_API_KEY', '<value>' );
define ( 'WC_OSA_ORDERS_INDEX_NAME', '<value>' );
define ( 'WC_OSA_ORDERS_PER_BATCH', '<value>' );
```

How does that sound?

**Comment by rayrutjes** [comment]
The suggested naming is perfect IMHO!

If you are doing a PR, please use conventional commit format: https://conventionalcommits.org/

That way, I'm sure to not steal your contribution again :/

**Comment by urlund** [comment]
@rayrutjes check PR #78 I hope it follows the conventional commit formats ;)

**Comment by urlund** [comment]
Fixed here #78

**Comment by rayrutjes** [comment]
Thanks for all the hard work @urlund and your patience.

You can now use constants in v1.11.0.
